docs/_posts/2017-09-26-react-v16.0.md

```diff
@@ -0,0 +1,133 @@
+
+### Support for custom DOM attributes
+
+Instead of ignoring unrecognized HTML and SVG attributes, React will now [pass them through to the DOM](https://facebook.github.io/react/blog/2017/09/08/dom-attributes-in-react-16.html). This has the added benefit of allowing us to get rid of most of React's attribute whitelist, resulting in reduced file sizes.
+
+### Reduced file size
+
+Despite all these additions, React 16 is actually **smaller** compared to 15.6.1!
+
+* `react` is 5.3 kb (2.2 kb gzipped), down from 20.7 kb (6.9 kb gzipped).
+* `react-dom` is 103.7 kb (32.6 kb gzipped), down from 141 kb (42.9 kb gzipped).
+* `react` + `react-dom` is 109 kb (34.8 kb gzipped), down from 161.7 kb (49.8 kb gzipped).
+
+That amounts to a combined **32% size decrease compared to the previous version (30% post-gzip)**.
+
+The size difference is partly attributable to a change in packaging. React now uses [Rollup](https://rollupjs.org/) to create flat bundles for each of its different target formats, resulting in both size and runtime performance wins. The 
+
+### New core architecture
+
+React 16 is the first version of React built on top of a new core architecture, codenamed "Fiber." You can read all about this project over on Facebook's engineering blog (TODO: link). (Spoiler: we rewrote React!)
+
+Most of the features in this release, like error boundaries and fragments, were made possible by the rewritten core. Over the next few releases, you can expect more new features as we begin to unlock the full potential of React.
+
+Perhaps the most exciting area we're working on is **async rendering**—a strategy for cooperatively scheduling rendering work by periodically yielding execution to the browser. The upshot is that, with async rendering, apps are more responsive because React avoids blocking the main thread.
+
+See this demo for an early peek at the types of problems async rendering can solve. TODO: link to demo.
+
+We think async rendering is a big deal, and represents the future of React. To make migration to v16.0 as smooth as possible, we're not enabling any async features yet, but we're excited to roll this out in the coming months. Stay tuned!
+
+## Upgrading
+
+Although React 16 includes significant internal changes, in terms of upgrading, you can think of this like any other major React release. We've been serving React 16 to Facebook and Messenger.com users since earlier this year, and we released several beta and release candidate versions to flush out additional issues. With minor exceptions, **if your app runs in 15.6 without any warnings, it should work in 16.**
+
+### Deprecations
```

Comment:
Maybe worth saying "New Deprecations" to make it clear these aren't deprecations we removed